debian/control

```diff
@@ -28,7 +28,6 @@ Build-Depends: debhelper (>= 10),
                libboost-dev,
                libspdk-dev,
                libipmctl-dev,
-               libraft-dev (= 0.8.1-1389.g304b19b),
```

Comment:
no action requested, just an observation/questions

The existing daos master code appears to, on this line (for ubuntu/debian builds), refer to raft development packages based upon daos-stack/raft git commit 304b19b:

commit 304b19b
Author: Brian J. Murrell <brian.murrell@intel.com>
Date:   Tue Sep 14 23:10:42 2021 -0400
Convert packaging/ to a submodule (#52)

Meanwhile, the daos.spec (RPM spec file) specifies a dependency on daos-raft-devel = 0.8.1
that if I am looking at the latest successful build artifacts in daos-stack/raft, is based on this later commit
commit cb47e29
Author: Brian J. Murrell <brian.murrell@intel.com>
Date:   Fri Oct 15 12:52:21 2021 -0400
Revert packaging/ as a submodule (#56)
    
cb47e29 also follows an intermediate landing:
commit 7ceb934
Author: David Quigley <david.quigley@intel.com>
Date:   Fri Sep 24 13:56:23 2021 -0600
DAOS-7661 bandit: Resolve Existing Issues found by Bandit (#55)

I'm not 100% sure what that may mean for current daos master builds. Is it
- daos ubuntu/deb package builds are based on raft with slightly older commit 304b19b
- while daos RPM package builds are based on raft with newer commit cb47e29 (and 7ceb934)?
- and daos builds from source are based on raft with the same newer commit cb47e29 ?

And with this PR daos master will now refer to the latest (as of today) git commit in daos-stack/raft repo:
commit a47758c
Author: Li Wei <wei.g.li@intel.com>
Date:   Wed Oct 20 09:25:09 2021 +0800
DAOS-4689 contrib: Add CLinkedListQueue (#54)

Reply:
Right, this is one of the problems I mentioned in the new proposal: "daos.rpm and daos.deb of the same build may use differnet raft commits." I can only think of setting the spec file line also to a specific package like the control file (Brian told me that for some unknown reason if we put just 0.8.1 in the control file, it somehow wouldn't work). That, however, seems not something Brian wants (see for example https://github.com/daos-stack/daos/commit/7ec7f6c254ea422c66409fe314db76a46a1a5c8f#diff-157dea0c50d0811d0acdc68394f79827b7e57044a6b765c2f2cfead1150226efR87).

Reply:
> (Brian told me that for some unknown reason if we put just 0.8.1 in the control file, it somehow wouldn't work)

By the way, I tried changing the raft version requirement in the control file to "0.9.0" today. It indeed was unable to find the 0.9.0 Debian raft packages.